src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -877,11 +883,26 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 			return this.renderToolInvocation(content, context, templateData);
 		} else if (content.kind === 'working') {
 			return this.renderWorkingProgress(content, context);
+		} else if (content.kind === 'undoStop') {
+			return this.renderUndoStop(content);
 		}
 
+		// todo@connor4312/roblourens: should this throw or renderNoContent to avoid progressive render thrashing?
```

Comment:
@roblourens what do you think?

Reply:
Oh, sorry for the break! If this method returns undefined, it tries to render that part forever? Your renderNoContent makes sense. Actually there's a couple cases already that render an empty dom element, like when a progress message hides.